examples/pytorch/nlp/huggingface_models/language-modeling/pruning/eager/run_clm_sparsegpt.py

```diff
@@ -621,7 +514,8 @@ def group_texts(examples):
         pruning_configs=[
             {
                 "pruning_type": "sparse_gpt",
-                "op_names": [".attn", "_proj", ".fc", "key", "dense", "_h"],
+                "op_names": [".*"],
+                "excluded_op_names": ["lm_head", "embed_out"],
```

Comment:
is it ok to remove this? Sparsegpt should only handle layers in block

Reply:
I've tested it, everything looks fine.